src/commands/dev-env-import-sql.ts

```diff
@@ -78,15 +88,14 @@ export class DevEnvImportSQLCommand {
 			const expectedDomain = `${ this.slug }.${ lando.config.domain }`;
 			await validateSQL( resolvedPath, {
 				isImport: false,
-				skipChecks: [],
+				skipChecks: isMyDumper ? [ 'dropTable', 'dropDB' ] : [],
 				extraCheckParams: { siteHomeUrlLando: expectedDomain },
 			} );
 		}
 
 		const fd = await fs.promises.open( resolvedPath, 'r' );
-		const importArg = [ 'db', '--disable-auto-rehash' ].concat(
-			this.options.quiet ? '--silent' : []
-		);
+		const importArg = this.getImportArgs( dumpDetails );
```

Comment:
Extracted to a function to satisfy `eslint` complexity limit - which I totally agree with.